src/Workleap.Extensions.Mongo.Analyzers.Tests/IndexAttributeUsageAnalyzerTests.cs

```diff
@@ -3,11 +3,13 @@
 public class IndexAttributeUsageAnalyzerTests : BaseAnalyzerTest<IndexAttributeUsageAnalyzer>
 {
     private const string TestClassName = "MyWorker";
+    
+    private const string TestMethodName = "DoSomething";
 
     [Theory]
     [InlineData("[IndexedBy(\"PrimaryKey\")]")]
     [InlineData("[NoIndexNeeded(\"Default index used\")]")]
-    public async Task Given_IndexAttribute_When_Analyze_Then_No_Diagnostics(string attribute)
+    public async Task Given_IndexAttribute_On_Class_When_Analyze_Then_No_Diagnostics(string attribute)
```

Comment:
Could we also add a test case with a local function? Had not thought about this one the first time